packages/imperative/src/imperative/src/plugins/utilities/npm-interface/update.ts

```diff
@@ -12,17 +12,18 @@
 import { PMFConstants } from "../PMFConstants";
 import { Logger } from "../../../../../logger";
 import { getPackageInfo, installPackages } from "../NpmFunctions";
+import { INpmRegistryInfo } from "../../doc/INpmRegistryInfo";
 
 /**
  * @TODO - allow multiple packages to be updated?
  * Common function that abstracts the update process.
  *
  * @param {string} packageName A package name. This value is a valid npm package name.
  *
- * @param {string} registry The npm registry.
+ * @param {INpmRegistryInfo} registryInfo The npm registry to use.
  *
  */
-export async function update(packageName: string, registry: string) {
+export async function update(packageName: string, registryInfo: INpmRegistryInfo) {
```

Comment:
Kind of nice to be able to do this 😋 

These are only exported in the context of the imperative package, but the `packages/imperative/src/imperative/index.ts` only exports the PMF and the lifeCycle classes from the plugins folder 😋

Reply:
That's right, AFAICT these methods are only used internally and not exported at the top-level, so these changes are non-breaking 😋